src/vs/monaco.d.ts

```diff
@@ -6241,7 +6241,7 @@ declare namespace monaco.languages {
 
 	/**
 	 * An event emitted when a language is associated for the first time with a text model or
-	 * whena language is encountered during the tokenization of another language.
```

Comment:
This file is automatically generated. Instead try finding the original comment in a normal `.ts` file and fix it there

Reply:
Thanks @mjbvz !

I found the original file, and fix it in [ddc357c](https://github.com/microsoft/vscode/pull/189449/commits/ddc357ca1cf09bf82c096e6cce63210e5ca3ddf7).

Could you check it again please?